py-wacz/wacz/util.py

```diff
@@ -9,4 +9,33 @@ def support_hash_file(data):
 
 
 def now():
+    """Returns the current time"""
     return tuple(datetime.datetime.utcnow().timetuple()[:6])
+
+
+def validateJSON(jsonData):
+    """Attempts to validate a string as json"""
+    try:
+        json.loads(jsonData)
+    except ValueError as err:
+        return False
+    return True
+
+
+def validate_passed_pages(passed_pages):
+    """Validates that a passed pages.jsonl file has the correct header and is valid json"""
+    if "format" not in json.loads(passed_pages[0]).keys():
+        print("The header of the jsonl file is missing the format key")
+        return 0
+    if "id" not in json.loads(passed_pages[0]).keys():
+        print("The header of the jsonl file is missing the id key")
+        return 0
+    if "title" not in json.loads(passed_pages[0]).keys():
```

Comment:
The title is also optional, can be detected if along with `text` if using text extraction, otherwise can be omitted. Will default to URL when reading.